_observing-your-data/sso.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Simple Schema for Observability
+nav_order: 120
+---
+
+# Simple Schema for Observability
+Introduced 2.6
+{: .label .label-purple }
+
+OpenSearch 2.6 introduced a standardization for conforming to a common and unified observability schema: Simple Schema for Observability (SSO). [Observability]({{site.url}}{{site.baseurl}}/observing-your-data/index/) is a collection of plugins and applications that let you visualize data-driven events by using Piped Processing Language (PPL) to explore and query data stored in OpenSearch.
+
+With the schema in place, Observability tools can ingest, automatically extract, and aggregate data and create custom dashboards, making it easier to understand the system at a higher level.
+
+SSO is inspired by both [OpenTelemetry](https://opentelemetry.io/docs/) and the Elastic Common Schema (ECS) and uses [ECS events](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_cwe_events.html) for logs with the OpenTelemetry metadata.
+
+Alerts will be supported in a future release.
+{: .note }
+
+## Use cases
+
+Use cases for SSO include:
+
+* Ingesting Observability data from different data types.
+* Moving from proprietary configurations that are non-transferable to a consolidated, sharable Observability solution that allows users to ingest and display an analysis of any type of telemetry data from any type of provider.
```

Comment:
In this bullet and the next one, confirm that the capitalized instances of "Observability" are actually references to the proper noun and not just generic references to the concept of observability, which should be lowercase.